kiva/ps.py

```diff
@@ -199,7 +210,7 @@ def device_draw_image(self, img, rect):
 
         Requires the Python Imaging Library (PIL).
         """
-        from kiva.compat import pilfromstring, piltostring
+        from kiva.compat import pilfromstring, piltostring, Image as PilImage
```

Comment:
It's used in the function but never imported in the module.(Line 249). I can move the import at the top of the module, but that could have more important impact.